src/mvPlotting.cpp

```diff
@@ -2655,6 +2925,16 @@ DearPyGui::set_configuration(PyObject* inDict, mvCustomSeriesConfig& outConfig)
 	if (PyObject* item = PyDict_GetItemString(inDict, "y2")) { (*outConfig.value)[3] = ToDoubleVect(item); }
 	if (PyObject* item = PyDict_GetItemString(inDict, "y3")) { (*outConfig.value)[4] = ToDoubleVect(item); }
 	if (PyObject* item = PyDict_GetItemString(inDict, "tooltip")) { outConfig.tooltip = ToBool(item); }
+
+	// helper for bit flipping
+	auto flagop = [inDict](const char* keyword, int flag, int& flags)
+	{
+		if (PyObject* item = PyDict_GetItemString(inDict, keyword)) ToBool(item) ? flags |= flag : flags &= ~flag;
+	};
+
+	// flags
+	flagop("no_fit", ImPlotItemFlags_NoFit, outConfig.flags);
+	flagop("no_legend", ImPlotItemFlags_NoLegend, outConfig.flags);
```

Comment:
Do we really need this new argument? An empty label (`label=""` or just omitting it) has the same effect (even handled by the same `if` within ImPlot), and it works for all other kinds of series, too. Is there anything special in `custom_series` that it needs a separate flag for this?

Reply:
It's just that those `ImPlotItemFlags` are common between all the plots, and I was putting it there to remember to apply it to also the other plots, but I wanted to inform better about it (it's one of the things written in the `TODO.md`
Also I liked the idea to stick as most as we could to the original paradigm... Should I remove it anyway?

Reply:
Well, it doesn't harm to have it, at least, so I don't really care. If I were implementing it, I'd keep changes to the minimum and would just rely on the existing `label=""` approach. However, if you think it adds value to have the `no_legend` flag, it's fine to keep it, as long as it is present in all series where it's applicable. Leaving it to your choice ;).

Reply:
I mean these parameters are applicable to all of the series but I don't know how.. I'd really like for example to implement the `no_fit` for also the others, but that would mean introduce the `ImPlot::BeginItem` for each kind of plot... Do you think there are other ways?

Reply:
`no_fit` seems to be a different thing, my initial concern was about `no_legend` that seems to be odd if added on only one series, because for all others we can use `label=""`. Well, I have to admit that to understand `no_fit` and `ImPlot::BeginItem`, I'd have to dig deeper into ImPlot than I can afford at the moment - `BeginItem` is not that documented :).

How about that we remove `no_legend` for now and call it a day? (keep `no_fit` since it doesn't seem to have any alternatives).

Reply:
Yeah, sounds good then!